.github/workflows/publish.yml

```diff
@@ -53,10 +53,10 @@ jobs:
       - uses: actions/checkout@v4
       
       - name: Build workspaces list
-        id: list_dirs
-        uses: ./.github/actions/get-workspaces
-        with: 
-          remove-internal-dockers: true
+        uses: ./.github/actions/get-members
```

Comment:
Same as in ci workflow + `--exclude` (if you accept my suggestion).
The list of directories to exclude could be saved in an env variable of the workflow, this way why and where some folders need to be excluded is made clear by the context.